api/src/auth/drivers/openid.ts

```diff
@@ -173,8 +173,8 @@ export class OpenIDAuthDriver extends LocalAuthDriver {
 
 		const userPayload = {
 			provider,
-			first_name: userInfo['given_name'],
-			last_name: userInfo['family_name'],
+			first_name: userInfo[(this.config['firstNameKey']) ? this.config['firstNameKey'] : 'given_name'],
+			last_name: userInfo[(this.config['lastNameKey']) ? this.config['lastNameKey'] : 'family_name'],
```

Comment:
```suggestion
			first_name: userInfo[this.config['firstNameKey'] ?? 'given_name'],
			last_name: userInfo[this.config['lastNameKey'] ?? 'family_name'],
```

Reply:
I forgot that the double `??` existed. It is so confusing switching between multiple languages haha